scripts/add-mentor.js

```diff
@@ -114,10 +114,10 @@ async function addToMentorsList(mentor) {
     await gitFunctions.branchProcess(answers);
     await gitFunctions.commitProcess(answers);
     await gitFunctions.pushProcess(answers);
-    console.log('\nMentor added.Please now create a PR for finish the process. Thanks!');
+    console.log('\nMentor added. Please create a PR to finish the process of adding a new mentor. Thanks!');
```

Comment:
It's belong to a different PR, don't you think? :)

Reply:
Oh yeah, thats true, I just couldn;t let it go I guess :smile:

Reply:
Don't forget to open a new PR for this.
Thanks!